### DISCUSSION TITLE
Worker in Animate debug mode

### DISCUSSION THREAD

**Discussion by MalacTheLittle** [airsdk/Adobe-Runtime-Support#2214] [Q&A]
Is there a way to run Workers in Flash CC / Animate test/debug mode?
I managed to run it only in published app.

**Comment by itlancer** [comment]
Here bug about this issue: https://github.com/airsdk/Adobe-Runtime-Support/issues/399

**Reply by ajwfrost** [reply]
Yes, this seems to be a discussion about that bug. Copying some information in here:

When a connection is first made to the debugger, the runtime is set to pause so that the debugger can pass it any requests for breakpoints etc.. but with the worker, the debugger isn't sending anything so the worker is just halted waiting to be resumed..

We can work around this for now, but it does seem as if the debugger still isn't able to pick up information from this other connection, which means that traces aren't coming out from the worker to Animate. You can see them happening in that Scout window though. There will need to be an update to the debugger at some point I guess....

So -> there will be a bug fix so that the workers do actually start when they're running with an active debugger like from Animate -> but the debug information (traces) and breakpoints won't work in a secondary worker.

We can leave this item open as the topic to discuss the fact that FDB needs to be updated to cope with Workers...
